sass/_ResponsiveFacets.scss

```diff
@@ -1,22 +1,21 @@
 @import 'bourbon/bourbon';
 @import 'Variables';
-
 .CoveoSearchInterface.coveo-small-facets {
   .coveo-facet-dropdown-content {
     z-index: 20;
     margin: 0;
   }
-
   .coveo-facet-dropdown-header {
     @include order(2);
     @include defaultRoundedBorder();
     background-color: white;
-
     &.coveo-dropdown-header-active {
       z-index: 20;
     }
+    &.coveo-hidden-no-visible-facet {
```

Comment:
I think that's the wrong class name. You seem to be using `coveo-no-visible-facet` everywhere.